### ISSUE TITLE
Use of the term "bioenergy" in variable names 

### ISSUE THREAD

**Issue by kriegler100** [IAMconsortium/common-definitions#213]
Is there an inconsistency between using "biomass" and "bioenergy" in variable names? 

Example:  Carbon Removal|Geological Storage|Bioenergy 
Why not: Carbon Removal|Geological Storage|Biomass? 

Compare: Primary Energy|Biomass|w/ CCS

Or is there a systematic rule when to use bioenergy and when biomass? If not I suggest to use Biomass consistently. 

@volker-krey

**Comment by danielhuppmann** [comment]
Thank you @kriegler100, pinging @IAMconsortium/common-definitions-emissions and @IAMconsortium/common-definitions-land for comment

**Comment by strefler** [comment]
Agree it should be Biomass. I haven't found "Bioenergy" anywhere else, and would also stick with one consistent term

**Comment by danielhuppmann** [comment]
Implement in #208 for carbon removal and carbon capture.

There is still use of "Bioenergy" in "Agricultural Demand|Crops|Bioenergy" - should that also be revised?

**Comment by strefler** [comment]
Hm, good question. At the risk of contradicting my earlier comment, I think there it is actually adequate

**Comment by flohump** [comment]
I also agree with @strefler that `Crops|Bioenergy` is fine for Agricultural Demand, Production and Trade.

**Comment by strefler** [comment]
@danielhuppmann can this issue be closed now?
